modules/support-must-gather-targeted-collection.adoc

```diff
@@ -0,0 +1,46 @@
+:_mod-docs-content-type: CONCEPT
+[id="support-must-gather-targeted-collection_{context}"]
+= Reducing the size of must-gather output
+
+The `oc adm must-gather` command collects comprehensive cluster information. However, a full data collection can result in a large file that is difficult to upload and analyze and could result in timeouts. 
+
+To manage the output size and target your data collection for more effective troubleshooting, you can pass specific flags to the underlying `gather` script or scope the collection to particular resources.
+
+[id="excluding-all-pod-logs_{context}"]
```

Comment:
```suggestion
```
can't have L2s in a procedure; this should either be two modules (a concept and a procedure), or just one procedure with a short description